index.html

```diff
@@ -56,83 +63,139 @@ <h2>LET'S CONNECT</h2>
+                      2015
+                    </h4>
+                    <p>a daily selection of privately personalized reads; no accounts or
+                      sign-ups required.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>Ruby on Rails</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
+                <article class="card">
+                  <div class="card-container-1">
+                    <img src="images/snapshoot-portfolio-1.svg" alt="">
+                    <div class="card-content-1">
+                    <h3>Facebook 360</h3>
+                    <h4><span>facebook</span>&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      full stack Dev&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      2015
+                    </h4>
+                    <p>Exploring the future of media in facebook's first virtual Reality <br>
+                      app a place to discover and enjoy 360 points and videos on <br>
+                      Gear VR.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
+                <!--end of first card-->
+                <!--second card-->
+                <article class="card">
+                  <div class="card-container">
+                    <img src="images/snapshoot-portfolio-2.svg" alt="">
+                    <div class="card-content">
+                    <h3>tonic</h3>
+                    <h4><span>conopy</span>&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      back end dev&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      2015
+                    </h4>
+                    <p>a daily selection of privately personalized reads; no accounts or
+                      sign-ups required.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                  
+                </article>
+                <!--third card-->
+                <article class="card">
+          
+                  <div class="card-container-1">
+                    <img src="images/snapshoot-portfolio-3.svg" alt="">
+                    <div class="card-content-1">
+                    <h3>multi-post stories</h3>
```

Comment:
- [ ] [OPTIONAL] PLEASE it would be better if you use the right font provided in the Figma design